modular-docs-manual/content/topics/concept_reusing-a-module-in-an-assembly.adoc

```diff
@@ -0,0 +1,86 @@
+[#reusing-a-module-in-an-assembly]
+
+= Reusing a Module in an Assembly
+
+In some cases, a book will need to include the same module multiple times within the same assembly.
+How do you reuse modules within the same assembly without getting build errors for using duplicate anchor IDs?
+The answer is embedding a user-defined document attribute in the anchor ID name.
+
+Programming languages call this variable substitution, which is also know as variable interpolation.
+The AsciiDoc markup language is doing the same type of substitution with document attributes.
```

Comment:
Let's include a link here to the Asciidoctor manual that describes this: http://asciidoctor.org/docs/user-manual/#include-multiple